README.md

```diff
@@ -9,6 +9,8 @@
 Play HLS, DASH, and future HTTP streaming protocols with video.js, even where they're not
 natively supported.
 
+Included in video.js 7 by default!
```

Comment:
Might be cool to add a link to https://blog.videojs.com/video-js-7-is-here/#VHS-and-Video-js here